### PR TITLE
Remove hard dependency to jnr-fuse and changing version

### PR DESCRIPTION
**Pull request by anandmohan777**

### CODE REVIEW

File: fuseclient.sh [hunk 1]
```diff
@@ -1,12 +1,12 @@
 SERVICE_NAME="Fuse client"
-PATH_TO_JAR=target/ega-fuse-1.0-SNAPSHOT.jar
-PROCESSCNT=$(ps x |grep -v grep |grep -c "ega-fuse-1.0-SNAPSHOT.jar")
-PID=$(ps aux | grep "ega-fuse-1.0-SNAPSHOT.jar" | grep -v grep | awk '{print $2}')
+PATH_TO_JAR=target/ega-fuse-0.1.0.jar
```
Comment (jorizci) [inline fuseclient.sh]: You have the variable there but then you ignore it in the rest of the script, if you use that variable you won't need to change version number on so many places every time we change the version.

**Comment by jorizci** [comment]
Also, change documentation to refect the version of the jar.

File: fuseclient.sh (hunk 1 above)
Comment (anandmohan777) [reply fuseclient.sh]: Done

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #Java
 target/
+dependency-reduced-pom.xml
```
Comment (jorizci) [inline .gitignore]: Not needed?

Comment (anandmohan777) [reply .gitignore]: Maven generates it automatically every time we build the project.

**Pull request merged**